tools/hardhat/README.md

```diff
@@ -104,28 +104,24 @@ Tasks get run as if they are built in hardhat commands. Read tasks are executed
 (no account), e.g:
 
 ```bash
-npx hardhat --network mezo_testnet validatorPool:submitApplication --signer <validator address> --conspubkey <validator consensus address> --moniker <mezod moniker>
+npx hardhat --network testnet validatorPool:submitApplication --signer <validator address> --conspubkey <validator consensus address> --moniker <mezod moniker>
 ```
 
 ### How to Submit an application to Validator Pool
 
 Here is a step by step guide on how to submit an application to the PoA validator pool.
 
 ```bash
-# install dependencies
-cd tools/hardhat
-npm install
-
 # set your private key
 npx hardhat vars set MEZO_ACCOUNTS <your private key>
```

Comment:
I think this will be hard for validators as we don't explain how to get that key. At the same time, I think it's pretty easy to automate as well. The `v-kit.sh` scripts pipe all the commands to `mezod` anyway. It should be possible to do `v-kit.sh keys unsafe-export-eth-key <key-name>` to get the private key from keyring. One downside will be the fact that the script will prompt for the keyring password but this should be relatively easy to handle on the validator side.

Reply:
We can do:
```
./v-kit.sh shell
(in containers shell): mezod --home=$MEZOD_HOME keys unsafe-export-eth-key $KEYRING_NAME
```

We can also automate this to be fully non-interactive. Let us know if you want this.

Reply:
> We can also automate this to be fully non-interactive.

If this is something relatively easy to add, please do. 🙏

Reply:
ok, here's the change covering Docker flow when extracting validator's priv key. [1ff7a1c](https://github.com/mezo-org/validator-kit/pull/5/commits/1ff7a1c420dc1bd4ae4d55d67ecaf05018c887a5)